llvm/lib/Transforms/Utils/BuildLibCalls.cpp

```diff
@@ -320,6 +357,7 @@ bool llvm::inferNonMandatoryLibFuncAttrs(Function &F,
     Changed |= setWillReturn(F);
     Changed |= setDoesNotCapture(F, 1);
     Changed |= setOnlyReadsMemory(F, 0);
+    Changed |= setOnlyAccessesArgOrErrnoMem(F);
```

Comment:
Accesses locale?

Reply:
It shouldn't be inaccessible memory -- but if you say this is argmem + errnomem only you're saying that locale doesn't exist at all. We just shouldn't add memory attributes for things that access locale at all.

Reply:
Oh, that’s very true. Thanks for the clarification!